Stratis.Bitcoin/Features/Wallet/WalletManager.cs

```diff
@@ -413,7 +421,8 @@ public int LastBlockHeight()
 				return this.chain.Tip.Height;
 			}
 
-			return this.Wallets.Min(w => w.AccountsRoot.Single(a => a.CoinType == this.coinType).LastBlockSyncedHeight) ?? 0;
+            // todo: use max instead of min?
```

Comment:
I need to remove that todo. I found out after testing the next method as well. It makes sense but I would rename the method to GetSyncStartBlockHeight etc.